blog.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Unitary Fund</title>
+    <!-- CSS -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+
+    <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
+    <style type="text/css">
+        #mc_embed_signup{background:#fff; clear:left; font:14px "Lucida Console", Monaco, monospace; width:100%;}
+        #mce-EMAIL{padding: 11px; margin: 10px;}
+    </style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-19932157-4"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-19932157-4');
+    </script>
+</head>
+
+<body>
+<div class="layout">
+    <main id="content" class="content">
+        <section class="light">
+
+    <center>
+        <h1>Unitary Fund</h1>
+
+        <p class="tagline"><b>Because evolution is
+        <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)" target="_blank">unitary</a></b>.</p>
+
+    </center>
+
+    <p class="light">
+        <a href="#supporters">supporters</a> |
+        <a href="#team">team</a> |
+        <a href="#grants-made">grants made</a> |
+        <a href="#project-ideas">project ideas</a> |
+        <a href="#faq">faq</a> |
+        <a href="#blog">blog</a>
```

Comment:
```suggestion
        <a href="blog.html">blog</a>
```

This should link to a page that just has a list of links (like a blog index) and then your post should be on its own separate page linked from there. I can help set this up if you'd like.

Reply:
thanks. I am also changing the header menu that was suffering from the same issue.